web/styles/zulip.css

```diff
@@ -1506,6 +1506,9 @@ strong {
    on :focus. */
 .typeahead-menu {
     & li {
+        word-wrap: break-word;
+        max-width: 85vw;
```

Comment:
We don't need to set the max-width. You'll observe that even without this rule, the typeahead menu fits just within the viewport.

Reply:
Thanks @N-Shar-ma for the review. 
I think if we just set the word-wrap without applying the max-width property then a single word with many characters won't break to fit content. It might just be helpful for words with shorter lengths.

Reply:
Use `word-break`, not `word-wrap` as suggested [here](https://chat.zulip.org/#narrow/stream/9-issues/topic/typeaheads.20spill.20out.20of.20compose.20box/near/1712420)

Reply:
Ohh yes , thanks for your suggestions. Works fine by me.
However I would like to add a small point that it feels a little bit odd to look at the compose box having a little bit of margin outside while the typeaheads mention cover the complete screen.
Still seems a reasonable solution , so I would like to have your suggestions.

Reply:
I get your concern about the typeahead menu taking up the entire width, but since that issue is unrelated to spilling of long text options, I suggest tackling that in a different commit / PR, after the text spill issue here has been resolved.

Reply:
Okay Sure thanks , that seems reasonable.
I have updated the screenshots and PR accordingly , so you may have a look and suggest some changes if needed.